cmd/lint.go

```diff
@@ -267,6 +304,19 @@ func GetLintCommand() *cobra.Command {
 
 			RenderTimeAndFiles(timeFlag, duration, filesProcessedSize, filesProcessed)
 
+			if minScore > 10 {
```

Comment:
Does this mean we can't set the score lower than 10?

Reply:
You can never go lower than 10. It's hard coded into the core of the tool. 

https://github.com/daveshanley/vacuum/blob/main/statistics/statistics.go#L67